workflow/applications.py

```diff
@@ -103,7 +103,7 @@ def __init__(self, configuration: Configuration) -> None:
         self.do_bufrsnd = _base.get('DO_BUFRSND', False)
         self.do_gempak = _base.get('DO_GEMPAK', False)
         self.do_awips = _base.get('DO_AWIPS', False)
-        self.do_wafs = _base.get('DO_WAFS', False)
+        self.do_wafs = _base.get('WAFSF', False)
```

Comment:
Is there a todo to change this to `DO_WAFS` wherever it is being used so we can change it back to match the other settings?